src/__tests__/Config.test.js

```diff
@@ -5,6 +5,8 @@ import { mkdtempSync } from 'fs';
 import * as path from 'path';
 import { getFixturePath } from 'jest-fixtures';
 
+jest.spyOn(console, 'error').mockImplementation(() => {});
```

Comment:
Why mock this instead of `logger`?

Reply:
... ... I Completely forgot which repo I was in. :man_facepalming: